src/postprocess/chi0.jl

```diff
@@ -108,7 +108,7 @@ function sternheimer_solver(basis, kpoint, Hk, ψk, ψnk, εnk, rhs, cgtol)
     # don't commute enough, and an oversolving of the linear
     # system can lead to spurious solutions
     rhs = Q(rhs)
-    δψnk = cg(J, rhs, Pl=FunctionPreconditioner(f_ldiv!), tol=cgtol / norm(rhs), verbose=true)
+    δψnk = cg(J, rhs, Pl=FunctionPreconditioner(f_ldiv!), tol=cgtol / norm(rhs), verbose=false)
```

Comment:
sorry about that, leftover debug

Reply:
Thought so ;)

Reply:
Converges pretty quick though 4 to 10 iterations. Not too bad.

Reply:
Yeah, it's supposed to be pretty quick, especially if you've got a few empty bands computed